packages/fxa-payments-server/src/components/ErrorMessage.scss

```diff
@@ -0,0 +1,26 @@
+@import '../../../fxa-content-server/app/styles/variables';
+
+.error-message-container {
+  background-color: $error-background-color;
+  border-radius: 4px;
+  color: $color-white;
+  padding: 10px;
+
+  p {
+    font-weight: 500;
+    margin: 0;
+    text-align: center;
+  }
+
+  + form {
+    margin-top: 12px;
+  }
+}
+
+.animate-fade-enter {
+  animation: fade-down-tt $short-transition;
```

Comment:
When I go through our CSS this quarter/in Q1, one thing I'll do is make our animation names a little more generic. We shouldn't need `-tt` here for "tooltip" for example, as now that I'm using it here, it feels like an awkward name.